pom.xml

```diff
@@ -61,6 +61,10 @@
 			<version>5.9.2</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.flywaydb</groupId>
+			<artifactId>flyway-mysql</artifactId>
+		</dependency>
```

Comment:
Let's not do mixed things in the PR. Big PRs are one of the main reasons, why software fails on production. I know we don't have this risk, but I want to teach you good practices.

PRs should be small and self-contained. They should have focused, singular goals:
- if there is a clean-up, let's not add new features
- if we're adding new code, let's not do major refactoring.
I recommend you look for guides and articles on how to structure and split PRs into small scopes. Another useful tactic it is to stack PRs on top of each other (useful while waiting for a review).

In this situation let's split into 2 PRs:
1. Clean-up, removing old code and controllers.
2. Adding Flyway integration.

Reply:
ok, Thanks! That makes totally sense